src/EventStore.ClusterNode/ClusterNodeOptions.cs

```diff
@@ -242,6 +242,9 @@ public class ClusterNodeOptions : IOptions
 
         [ArgDescription(Opts.StructuredLogDescr, Opts.DbGroup)]
         public bool StructuredLog { get; set; }
+		
+		[ArgDescription(Opts.AutomergeIndexesDescr, Opts.DbGroup)]
+        public bool AutomergeIndexes { get; set; }
```

Comment:
I would suggest `ManualIndexMerge` with a default of `False` instead of `AutoIndexMerge` so that the intent is clearer when setting it to `True`

Reply:
AutoIndexMerge=True seems to be clearer and it represents the current behavior. 
I got that option name from Greg from the initial discussion (check the related issue) and I prefer it. Can we double check another opinion on that before I make all the changes?